### PR TITLE
show more info from GitHub if a branch/tag has a problem [RUN-95]

### PR DESCRIPTION
**Pull request by jonmoter**
/cc @zendesk/runway 

### References
 - Jira link: https://zendesk.atlassian.net/browse/RUN-95

### Risks
 - Minimal, only effects user feedback on the "new deploy" page

### CODE REVIEW

**Comment by grosser** [comment]
:+1: sounds good, do you have a screenshot ?

File: app/assets/javascripts/deploys.js [hunk 1]
```diff
@@ -75,22 +76,39 @@ $(function () {
...
-          $ref_status_label.addClass("hidden");
-          tag_form_group.addClass("has-warning");
-        } else if(data.status == "success") {
-          $ref_status_label.addClass("hidden");
-          tag_form_group.addClass("has-success");
-        } else if(data.status == "failure" || data.status == "error") {
-          $ref_status_label.removeClass("hidden");
-          tag_form_group.addClass("has-error");
+        switch(data.status) {
+          case "success":
```
Comment (grosser) [inline app/assets/javascripts/deploys.js]: would be even better if js case statements weren't so shitty :D

**Comment by jonmoter** [comment]
Screenshot looks like this:

samson

**Comment by grosser** [comment]
sweet :)

**Pull request merged**